src/month.rs

```diff
@@ -201,7 +201,7 @@ impl Months {
 }
 
 /// An error resulting from reading `<Month>` value with `FromStr`.
-#[derive(Clone, PartialEq)]
+#[derive(Clone, PartialEq, Eq)]
```

Comment:
+`Copy` while you're at it? Or don't you want to make this promise in case of possible future changes?

Reply:
I'm just fixing up clippy stuff here, don't want to muddle it with unrelated changes.